contentwarehouse/snippets/list_documents_folder_sample.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START list_documents_folder_sample]
+import sys
+from google.cloud import contentwarehouse
+# TODO(developer): Uncomment these variables before running the sample.
+# project_number = "YOUR_PROJECT_NUMBER"
+# location = "us" # Format is 'us' or 'eu'
+# user_id = "user:xxxx@example.com" # Format is "user:xxxx@example.com"
+
+def create_folder(project_number: str, location: str, user_id: str) -> contentwarehouse.Document:
+
+    # Create a Schema Service client
+    document_schema_client=contentwarehouse.DocumentSchemaServiceClient()
+
+    # The full resource name of the location, e.g.:
+    # projects/{project_number}/locations/{location}
+    parent=document_schema_client.common_location_path(
+        project=project_number, location=location
+    )
+
+    # Define Folder Schema Request
+    create_folder_schema_request=contentwarehouse.CreateDocumentSchemaRequest(
+        parent=parent,
+        document_schema=contentwarehouse.DocumentSchema(
+            display_name="Test Folder Schema ",
+            document_is_folder = True,
+        ),
+    )
+
+    # Create a Folder Schema
+    folder_schema=document_schema_client.create_document_schema(
+        request=create_folder_schema_request
+    )
+
+    # Create a Document(Folder) Service client
+    folder_client=contentwarehouse.DocumentServiceClient()
+
+    # Define Folder
+    folder=contentwarehouse.Document(
+        display_name="My Test Folder",
+        document_schema_name=folder_schema.name,
+    )
+
+    # Define Request to create Folder
+    create_folder_request=contentwarehouse.CreateDocumentRequest(
+        parent=parent,
+        document=folder,
+        request_metadata=contentwarehouse.RequestMetadata(
+            user_info=contentwarehouse.UserInfo(id=user_id))
```

Comment:
each list item laid out vertically should always end with a comma - some languages don't do this but this is preferred style for Python.
```suggestion
            user_info=contentwarehouse.UserInfo(id=user_id)),
```